api/src/index.js

```diff
@@ -24,6 +28,58 @@ apiRouter.get("/", async (req, res) => {
 // This nested router example can also be replaced with your own sub-router
 apiRouter.use("/nested", nestedRouter);
 
+apiRouter.get("/future-meals", async (req, res) => {
+  try {
+    const meals = await knex.raw("SELECT * FROM Meal WHERE `when` > NOW()");
```

Comment:
how about refactoring those endpoints now that you know how to use knex functions? ;)